### ISSUE TITLE
Why emdb test dataset use estimated K_fullimg?

### ISSUE THREAD

**Issue by hengyuan-zhang-0** [zju3dv/GVHMR#73]
Hi,

I’ve encountered an issue when rendering the in-cam mesh using the estimated K_fullimg. The mesh does not seem to align properly with the human figure, whereas using the ground truth (GT) K_fullimg works much better. Could you help me understand why the estimated K_fullimg is being used for evaluation? I assume that other datasets also use the ground truth K_fullimg.

Looking forward to your response.

**Comment by zehongs** [comment]
Hi Hengyuan,
1) What K_fullimg are you using for estimating in-cam mesh? It should be the same one used in visualization.
2) The results should be similar, I don't remeber why I was using the estimated K_fullimg.

**Comment by hengyuan-zhang-0** [comment]
Hi zehongs,
(1) I am trying to find the original dataset images that correspond to the processed annotations provided by GVHMR. To do this, I need to perform some visualization to check the correspondence.
(2) In this experiment, I first used the estimated K_fullimg in P1, which resulted in P2. Then, I used the K_fullimg = label["K_fullimg"], and the results were P3.


<img width="841" alt="Image" src="https://github.com/user-attachments/assets/39c70229-5fb6-440b-bee4-2187523bc2b3" /> 

<img width="720" alt="Image" src="https://github.com/user-attachments/assets/6dfd7e03-0a58-4fca-b46c-573c5d7f4214" /> 

<img width="998" alt="Image" src="https://github.com/user-attachments/assets/b42d1a2e-bfa6-4d4a-b820-797206087253" />

**Comment by zehongs** [comment]
https://github.com/zju3dv/GVHMR/blob/a0cbfa59b9b20a4ab2f5348475e9c8752762005a/hmr4d/dataset/emdb/emdb_motion_test.py#L94-L111

https://github.com/zju3dv/GVHMR/blob/a0cbfa59b9b20a4ab2f5348475e9c8752762005a/hmr4d/model/gvhmr/callbacks/metric_emdb.py#L223

I'm sorry for the confusion. I think the video has been resized to a smaller size. Please double-check the source of K.

**Comment by hengyuan-zhang-0** [comment]
Thank you for your reply. I will check this. Additionally, I have a few questions: For the training data, 
(1)  is the bbox calculated following Bedlam's projected bbox method, or  based on YOLOv8's bbox? (2) Is the bbox using base_enlarge = 1.2, or was bbox augmentation applied? (3)  are   bboxs  smoothed?

**Comment by zehongs** [comment]
(1) We use the ground truth bounding boxes from BEDLAM. Please note that you are responsible for carefully handling any border effects.
(2) Only `base_enlarge = 1.2`.
(3) No.

**Comment by hengyuan-zhang-0** [comment]
Hi Zehong,

(1) I added some datasets for training; however, they caused the performance of emdb2's wa2_mpjpe and waa_mpjpe metrics to degrade significantly. 
Subsequently, I standardized the world coordinate systems.
For example, the original Bedlam uses a right-handed coordinate system with the Y-down, so I converted it to a Y-up coordinate system and face z+. 
Despite this standardization, the performance of emdb2's wa2_mpjpe and waa_mpjpe metrics still remains poor. 
(2) Could you help me understand why this might be happening? Have I overlooked any important techniques?

**Comment by zehongs** [comment]
Hi hengyuan
The definition of the coordinate system is important because we compute the GV system based on it.
What does the performance curve look like during finetuning? As mentioned in the README, we do not apply post-processing during training, so the performance might drop for this reason.

**Comment by hengyuan-zhang-0** [comment]
Hi，zehong
(1) I find the performance on emdb2 is poor while the performance on other datasets is normal.
(2) And I also process the bedlam's camera pose to transform the smpl-params-camera to  smpl-params-world in y-up, gravity-vector=(0,-1,0) coordiante (since the original annot in bedlam is y-down) 
(3) I wonder what additional data processing steps I need to perform. I have already computed T_c2w from T_w2c (camera_pose), then multiplied R_c2w by R = torch.tensor([[1, 0, 0], [0, -1, 0], [0, 0, -1]]), and rotated t_c2w by 180 degrees around the x-axis.

<img width="622" height="289" alt="Image" src="https://github.com/user-attachments/assets/ff360441-3686-4636-81ba-85b6bedf934a" />

<img width="623" height="257" alt="Image" src="https://github.com/user-attachments/assets/791dd09b-76f8-4de6-9a2d-cfb0812f49d6" />

**Comment by zehongs** [comment]
3DPW/pa_mpjpe only evaluates the in-cam performance, which is not related to the GV definition. Could you show me the RICH/wa2_mpjpe results ?
By the way, I'm not confident that the geometric transformation is correct. I would suggest doing some visualization.
